src/lmms_engine/parallel/expert_parallel/qwen3_moe/style.py

```
@@ -0,0 +1,127 @@
+            for expert_idx in range(start_idx, end_idx):
+                new_experts.append(mod.experts[expert_idx])
+
+            # Update the module list with the new experts
+            setattr(mod, "experts", new_experts)
+
+    # The token all to all dispatch will be handled in the ops
+    # Check the lmms_engine/models/qwen3_moe/qwen3_moe_ops.py
+    # Since the expert module is a module list, we don't define the token
+    # dispatch here
+    @staticmethod
+    def _input_fn(input_layouts, desired_input_layouts, mod, input, device_mesh):
```

Ruff (0.12.2)

Unused static method argument: `mod`

(ARG004)